locale/cs/LC_MESSAGES/clipboard-indicator.po

```diff
@@ -59,27 +58,27 @@ msgstr "Následující položka"
 
 #: prefs.js:119
 msgid "History Size"
-msgstr "Velikost historie"
+msgstr "Hloubka historie"
 
 #: prefs.js:124
 msgid "Refresh Interval (ms)"
-msgstr "Interval obnovení (ms)"
+msgstr "Interval obnovování (ms)"
 
 #: prefs.js:129
 msgid "Preview Size (characters)"
 msgstr "Velikost náhledu (znaky)"
 
 #: prefs.js:134
 msgid "Max cache file size (kb)"
-msgstr "Maximální velikost mezipaměti (kb)"
+msgstr "Omezit velikost mezipaměti na nejvýše (KB)"
 
 #: prefs.js:139
 msgid "Disable cache file"
-msgstr "Zakázat soubor mezipaměti"
+msgstr "Nepoužívat mezipaměť (soubor)"
```

Comment:
"Nepoužívat soubor mezipaměti" sounds better.

Reply:
It is don't use what? Cache. Detail – implemented as what? File.
@Tudmotu what is the point of communication with user there?  Is it about store data only in RAM (so available only during desktop session) instead in file (where it persist across sessions)?
In that case it could be there "Don't save to disk (no persistence)" maybe?
That "Disable cache file" may be too much technical for average user (as kind mainly targeted by Gnome project)…